/*
 * controller_core.h
 *
 *  Created on: May 2, 2012
 *      Author: Michael Anderson
 */

#ifndef CONTROLLER_CORE_H_
#define CONTROLLER_CORE_H_

#include <ac_motor_torque/controller_common.h>
#include <atrias_control/core_library.h>

double progress = 0.5;
bool goingUp = true;

#endif /* CONTROLLER_H_ */
